### PR TITLE
adding launch folder and the launch.py files and camera .xml

### PR DESCRIPTION
**Pull request by allegrocantabile**
This change adds the laser.launch.py, ov5645.launch.xml, and robot.launch.py for MELM

### CODE REVIEW

File: melm_bringup/launch/laser.launch.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            {'port_baudrate': 921600},
+            {'laser_scan_dir': True},
+            {'enable_angle_crop_func': False},
+            {'enable_box_crop_func': True},
+            {'angle_crop_min': 175.0},
+            {'angle_crop_max': 210.0},
+            {'x_crop_min': -0.30},
+            {'x_crop_max': 0.20},
+            {'y_crop_min': -0.09},
+            {'y_crop_max': 0.09},
```
Comment (bperseghetti) [inline melm_bringup/launch/laser.launch.py:46]: We should consider adjusting these to match the dimensions of the MELM like we did for the B3RB

File: melm_bringup/launch/ov5645.launch.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  <arg name="device" default="/dev/video3"/>
+  <arg name="fps" default="30"/>
+  <arg name="width" default="320"/>
+  <arg name="height" default="240"/>
+
+  <node namespace="" name="ov5645" pkg="gscam" exec="gscam_node" output="screen">
+    <param name="camera_name" value="camera"/>
+    <param name="camera_frame" value="camera_link_optical"/>
+    <param name="camera_info_url" value="package://melm_bringup/config/ov5645.ini"/>
+    <param name="gscam_config" value="v4l2src device=$(var device) ! video/x-raw,format=BGRx,framerate=$(var fps)/1,width=$(var width),height=$(var height) ! imxvideoconvert_g2d rotation=2 ! video/x-raw,format=BGRx,framerate=$(var fps)/1,width=$(var width),height=$(var height) ! videoconvert ! video/x-raw,format=BGR"/>
```
Comment (bperseghetti) [inline melm_bringup/launch/ov5645.launch.xml:11]: If you wanted to do a rotation it would happen on this line using gstreamer command for it.

**Review by bperseghetti — APPROVED** [review]

**Comment by bperseghetti** [comment]
@allegrocantabile great contribution, thanks for opening this PR, I added a few comments in the PR that I think are good to consider but can be added later.

File: melm_bringup/launch/ov5645.launch.xml (hunk 2 above)
Comment (allegrocantabile) [reply melm_bringup/launch/ov5645.launch.xml:11]: I will work on camera image rotation after doing some tests.  
I was playing around with the values in rotation parameter:
imxvideoconvert_g2d rotation= 0

but that caused the camera image to disappear.  I need to do some more tests on my side before finalizing it.  I will update once the test is completed.

File: melm_bringup/launch/laser.launch.py (hunk 1 above)
Comment (allegrocantabile) [reply melm_bringup/launch/laser.launch.py:46]: will do for the next upload.  I need to fix the camera image, so I'd upload this along with the fix.

**Pull request merged**